packages/frontend-api/apiary.apib

```diff
@@ -0,0 +1,1132 @@
+                         "priceWithVat": "2.00",
+                         "priceWithoutVat": "2.00",
+                         "vatAmount": "0.00"
+                     },
+                     "images": [
+                         {
+                             "type": null,
+                             "position": null,
+                             "size": "default",
+                             "url": "http://127.0.0.1:8000/content/images/payment/default/55.jpg",
+                             "width": null,
+                             "height": 20
+                         },
+                         {
+                             "type": null,
+                             "position": null,
+                             "size": "original",
+                             "url": "http://127.0.0.1:8000/content/images/payment/original/55.jpg",
+                             "width": null,
+                             "height": null
+                         }
+                     ],
+                     "transports": [
+                         {
+                             "uuid": "edbafa26-9306-46ed-b1e8-d1570f072832",
+                             "name": "Czech post"
+                         }
+                     ]
+                 },
+                 {
+                     "uuid": "76145869-900e-4051-a7be-2e0d1796cd6b",
+                     "name": "Cash",
+                     "description": null,
+                     "instruction": null,
+                     "position": 2,
+                     "price": {
+                         "priceWithVat": "0.00",
+                         "priceWithoutVat": "0.00",
+                         "vatAmount": "0.00"
+                     },
+                     "images": [
+                         {
+                             "type": null,
+                             "position": null,
+                             "size": "default",
+                             "url": "http://127.0.0.1:8000/content/images/payment/default/54.jpg",
+                             "width": null,
+                             "height": 20
+                         },
+                         {
+                             "type": null,
+                             "position": null,
+                             "size": "original",
+                             "url": "http://127.0.0.1:8000/content/images/payment/original/54.jpg",
+                             "width": null,
+                             "height": null
+                         }
+                     ],
+                     "transports": [
+                         {
+                             "uuid": "adecf603-f491-4ece-b049-8d0d86a1e48c",
+                             "name": "Personal collection"
+                         }
+                     ]
+                 }
+             ]
+         }
+     }
+
+## Login [/graphql]
+
+### POST [POST]
+
+Run a GraphQL query
+
+- Request (application/json)
+
+    mutation {
```

Comment:
I think so I might like a description of each method. In the beginning, the methods have only `Runs a graphQL query.` description. 

In this mutation (login) I would also like information about the lifetime of tokens, because I think so this pair of tokens will always be expired - the mocking probably will not affect, but we should probably write info. What do you think?

Reply:
updated